userguide/services.rst

```diff
@@ -2076,16 +2076,17 @@ UPS Configuration screen.
-   |                         |              | :guilabel:`Driver` becomes :guilabel:`Remote Host` when :guilabel:`UPS Mode` is set to *Slave*. The IP address of the  |
-   |                         |              | system configured as the UPS *Master* system. See this `post                                                           |
+   |                         |              | The text :guilabel:`Driver` becomes :guilabel:`Remote Host` when :guilabel:`UPS Mode` is set to *Slave*. The IP        |
+   |                         |              | address of the system configured as the UPS *Master* system. See this `post                                            |
    |                         |              | <https://forums.freenas.org/index.php?resources/configuring-ups-support-for-single-or-multiple-freenas-servers.30/>`__ |
    |                         |              | for more details about configuring multiple systems with a single UPS.                                                 |
    |                         |              |                                                                                                                        |
    +-------------------------+--------------+------------------------------------------------------------------------------------------------------------------------+
-   | Port / Hostname /       | drop-down    | :guilabel:`Port`: Enter the serial or USB port the UPS is plugged into (see :ref:`NOTE <UPS USB>`).                    |
-   | Remote Port             | menu         |                                                                                                                        |
-   |                         |              | :guilabel:`Port` becomes :guilabel:`Remote Port` when the :guilabel:`UPS Mode` is set to *Slave*. The open network     |
-   |                         |              | port number of the UPS *Master* system. The default port is *3493*.                                                    |
+   | Port / Remote Port      | drop-down    | :guilabel:`Port`: Enter the serial or USB port the UPS is plugged into (see :ref:`NOTE <UPS USB>`).                    |
+   |                         | menu         | When an :literal:`snmp` driver is selected, enter the IP address or hostname of the SNMP UPS device.                   |
+   |                         |              |                                                                                                                        |
+   |                         |              | The text :guilabel:`Port` changes to :guilabel:`Remote Port` when the :guilabel:`UPS Mode` is set to *Slave*.          |
+   |                         |              | It is the open network port number of the UPS *Master* system. The default port is *3493*.                             |
```

Comment:
`` The name of this field changes to :guilabel:`Remote Port` when :guilabel:`UPS Mode` is set to *Slave*. Enter the open network port number... ``